packages/hdwallet-metamask/src/adapter.ts

```diff
@@ -55,6 +44,17 @@ export class MetaMaskAdapter {
   }
 
   private async pairMetaMaskDevice(): Promise<core.HDWallet> {
```

Comment:
I don't see a need for this separate private function. Just make this the `public async pairDevice()` function.